data/categories.txt

```diff
@@ -1249,7 +1249,7 @@ amenity-place_of_worship
 en:place of worship|4temple|sights|U+1F64F|U+26EA
 ru:3храм|церковь|достопримечательность
 uk:3храм|церква|собор|пам’ятка|алтарь|мавзолей|пам’ятні місця
-de:4Tempel|Kirche|Dom|Sehenswürdigkeit
+de:Kirche|Synagoge|Moschee|4Tempel|Anbetungsstätte|Dom|Sehenswürdigkeit
```

Comment:
This translation should have only generic words, not related to any specific religion.
We have recently introduced religion-specific types but still not translated them all at the moment.